src/Routes/ImageManager/steps/imageSetDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import componentTypes from '@data-driven-forms/react-form-renderer/component-types';
+import { Text } from '@patternfly/react-core';
+import validatorTypes from '@data-driven-forms/react-form-renderer/validator-types';
+
+export default {
+  title: 'Image Set Details',
+  name: 'imageSetDetails',
+  nextStep: 'imageOutput',
+  fields: [
+    {
+      component: componentTypes.PLAIN_TEXT,
+      name: 'description',
+      label: (
+        <Text>
+          How would you like to identify this image later? What will it be used
+          for?
+        </Text>
+      ),
+    },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'image-name',
+      label: 'Image Name',
+      placeholder: 'Image Name',
+      validate: [{ type: validatorTypes.REQUIRED }],
```

Comment:
We need also to validate that the image contains alphanumeric characters and can contain `-` and `_`.

Reply:
:%s/the image/the image name/g